src/js/discharge-wizard/components/Breadcrumbs.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { buildMobileBreadcrumb, debouncedToggleLinks } from '../../utils/breadcrumb-helper';
+
+class Breadcrumbs extends Component {
+  componentDidMount() {
+    buildMobileBreadcrumb('va-breadcrumbs-discharge', 'va-breadcrumbs-discharge-list');
+
+    window.addEventListener('DOMContentLoaded', () => {
+      buildMobileBreadcrumb.bind(this);
+    });
+
+    window.addEventListener('resize', () => {
+      debouncedToggleLinks('va-breadcrumbs-discharge-list');
+      debouncedToggleLinks.bind(this);
+    });
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('DOMContentLoaded', () => {
+      buildMobileBreadcrumb.bind(this);
+    });
+
+    window.removeEventListener('resize', () => {
+      debouncedToggleLinks.bind(this);
+    });
+  }
```

Comment:
Does each application have its own breadcrumb component? Are there differences between them that can't be consolidated via props?